angular.json

```diff
@@ -35,6 +35,11 @@
                 "glob": "**/*.svg",
                 "input": "node_modules/ionicons/dist/ionicons/svg",
                 "output": "./svg"
+              },
+              {
+                "glob": "**/*",
+                "input": "node_modules/ngx-extended-pdf-viewer/assets/",
```

Comment:
Do we need all these assets? The folder is a massive 40MB and has many duplicate and unnecessary files (e.g. both .js and .min.js files, map files etc.)

Reply:
Also I noticed you can define the assets folder name in the using the `assetsFolder` option (https://pdfviewer.net/extended-pdf-viewer/options) - might be nice to bundle all these into their own folder instead of the main assets in case of any future filename conflicts